pkg/odo/cli/pipelines/webhook/create.go

```diff
@@ -0,0 +1,72 @@
+package webhook
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	"github.com/openshift/odo/pkg/log"
+	"github.com/openshift/odo/pkg/machineoutput"
+	"github.com/openshift/odo/pkg/odo/genericclioptions"
+	"github.com/spf13/cobra"
+
+	backend "github.com/openshift/odo/pkg/pipelines/webhook"
+	ktemplates "k8s.io/kubernetes/pkg/kubectl/util/templates"
+)
+
+const createRecommendedCommandName = "create"
+
+var (
+	createExample = ktemplates.Examples(`	# Create a new Git repository webhook 
+	%[1]s`)
+)
+
+type createOptions struct {
+	options
+}
+
+func newCreateOptions() *createOptions {
+	return &createOptions{}
+}
+
+// Run contains the logic for the odo command
+func (o *createOptions) Run() (err error) {
```

Comment:
You're not actually using the named return value in your code, you could probably remove it (same for delete).

Reply:
What?  id/ids returned by backend.Create()/backend.Delete()   The returned  values are certainly used.

Reply:
There's only one named return value here, `err error` and you don't use it.

https://github.com/rhd-gitops-example/odo/pull/78/files/72a200aee66bf233be76fbf5b922baf708ae6295#diff-a78fb2292082aa471554b1eb0abd84bfR35 you overwrite it as part of the short assignment.

And here https://github.com/rhd-gitops-example/odo/pull/78/files/72a200aee66bf233be76fbf5b922baf708ae6295#diff-a78fb2292082aa471554b1eb0abd84bfR53 you're explicitly returning `nil` which doesn't use it either.

Reply:
Did you see `err` is used here???
https://github.com/rhd-gitops-example/odo/blob/72a200aee66bf233be76fbf5b922baf708ae6295/pkg/odo/cli/pipelines/webhook/create.go#L37-L39

If you get here, you know `err` is nil a successful case.   Otherwise, you wouldn't be there.
https://github.com/rhd-gitops-example/odo/pull/78/files/72a200aee66bf233be76fbf5b922baf708ae6295#diff-a78fb2292082aa471554b1eb0abd84bfR53

Reply:
But that's not what it's doing, the short assignment will create a new assignment.

You can basically change `func (o *createOptions) Run() (err error)` to `func (o *createOptions) Run() error`

Reply:
Huh?  That's not what it is doing?
```
if err != nil { 
 	return fmt.Errorf("Unable to create webhook: %v", err) 
 } 
```
you mean it does not return an error?

Reply:
No, it's not returning the named return value.

It's returning the err from the short-assignment.

Reply:
Oh, you are talking about named return value. Wow!  Such a big deal!   I thought you mean it does not return an error.  Is using named return error wrong?   This signature was copy from other commands.   It is pretty staandard in odo command.